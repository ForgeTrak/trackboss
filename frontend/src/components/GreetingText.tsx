import React from 'react';
import {
    Center,
    Heading,
} from '@chakra-ui/react';

interface pageProps {
    name: string
}

// the below eslint-disable is needed because of the trailing space in the greeting
export default function Header(props:pageProps) {
    return (
        <Center>
            <Heading size="4xl">
                Hi, {/*eslint-disable-line*/}
                <Heading sx={{ display: 'inline' }} color="orange" size="4xl">
                    {props.name}
                </Heading>
            </Heading>
        </Center>
    );
}
