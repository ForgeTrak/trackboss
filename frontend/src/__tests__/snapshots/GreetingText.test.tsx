import React from 'react';
import renderer from 'react-test-renderer';
import GreetingText from '../../components/GreetingText';

it('renders correctly', () => {
    const greetingText = renderer.create(
        <GreetingText name="Martin" />,
    ).toJSON();
    expect(greetingText).toMatchSnapshot();
});
