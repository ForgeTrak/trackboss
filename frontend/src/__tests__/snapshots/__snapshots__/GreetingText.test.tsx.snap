// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-gmuwbf"
>
  <h2
    className="chakra-heading css-0"
  >
    Hi, 
    <h2
      className="chakra-heading css-11haukf"
    >
      Martin
      !
    </h2>
  </h2>
</div>
`;
